sdk/keyvault/azure-security-keyvault-certificates/src/samples/java/com/azure/security/keyvault/certificates/CertificateAsyncClientJavaDocCodeSnippets.java

```diff
@@ -95,15 +95,15 @@ public void getCertiificatePolicyCodeSnippets() {
             .subscriberContext(Context.of(key1, value1, key2, value2))
             .subscribe(policy ->
                 System.out.printf("Certificate policy is returned with issuer name %s and subject name %s %n",
-                    policy.getIssuerName(), policy.getSubjectName()));
+                    policy.getIssuerName(), policy.getSubject()));
```

Comment:
should this method be `getSubjectName` instead?

Reply:
nope, calling it subject across languages.